wagtail/admin/templates/wagtailadmin/reports/scheduled_pages.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'wagtailadmin/reports/base_page_report.html' %}
+{% load i18n %}
+{% block listing %}
+    {% include "wagtailadmin/reports/listing/_list_publish.html" %}
+{% endblock %}
+
+{% block no_results %}
+    <p>{% trans "No pages scheduled for publishing were found." %}</p>
+{% endblock %}
+
+{% block actions %}
+    {% if object_list.count > 0 %}
+        <div class="addbutton">
+            <a href="{% url 'wagtailadmin_scheduled_pages:publish_all_confirm' %}" class="button">{% trans 'Publish all scheduled pages' %}</a>
```

Comment:
What happens when the user filters the report by certain dates and wants to only publish those? currently this publishes *all* scheduled rather than the filtered ones. Not sore I'm on board

Reply:
I've removed "publish all" to lessen the danger of abuse /mistakes